django/test/runner.py

```diff
@@ -656,21 +680,8 @@ def teardown_test_environment(self, **kwargs):
     def suite_result(self, suite, result, **kwargs):
         return len(result.failures) + len(result.errors)
 
-    def _get_databases(self, suite):
-        databases = set()
-        for test in suite:
-            if isinstance(test, unittest.TestCase):
-                test_databases = getattr(test, 'databases', None)
-                if test_databases == '__all__':
-                    return set(connections)
-                if test_databases:
-                    databases.update(test_databases)
-            else:
-                databases.update(self._get_databases(test))
-        return databases
-
     def get_databases(self, suite):
-        databases = self._get_databases(suite)
+        databases = _get_databases(suite)
```

Comment:
What's the rationale behind these changes

Reply:
Ummm... thank you for notice that change. I forgot that I take out `_get_databases` outside of the class before. Previously, I need to access `_get_databases` function inside the init function (`_init_function`). Later on, I realize that I don't need to do that.